apps/api/src/app/events/usecases/parse-event-request/parse-event-request.command.ts

```diff
@@ -8,6 +8,7 @@ import {
 } from '@novu/shared';
 
 import { EnvironmentWithUserCommand } from '../../../shared/commands/project.command';
+import { ApiHideProperty } from '@nestjs/swagger';
```

Comment:
Needed to resolve the import order linting problem [here](https://cloud.nx.app/runs/x6j60dkQbF).

```suggestion
} from '@novu/shared';
import { ApiHideProperty } from '@nestjs/swagger';

import { EnvironmentWithUserCommand } from '../../../shared/commands/project.command';
```

Reply:
done